rero_ils/modules/holdings/api.py

```diff
@@ -241,6 +243,81 @@ def get_holding_loan_conditions(self):
             return ItemType.get_record_by_pid(
                 self.circulation_category_pid).get('name')
 
+    @property
+    def patterns(self):
```

Comment:
For me this is useless as `holding.get('patterns')` is not different to `holding.patterns`.